### ISSUE TITLE
environmentSetupScript capitalizes environment variable names

### ISSUE THREAD

**Issue by cetygamer** [microsoft/vscode-cmake-tools#1592]
### Brief Issue Summary

`environmentSetupScript` in `cmake-kits.json` converts all exported environment variable names to uppercase.

### Expected:

Variable names are preserved as is.

### Apparent Behavior:

Variable names are capitalized.

### CMake Tools Log

```
[main] Configuring folder: cmake-test 
[proc] Executing command: /usr/bin/cmake --no-warn-unused-cli -DCMAKE_EXPORT_COMPILE_COMMANDS:BOOL=TRUE -DCMAKE_BUILD_TYPE:STRING=Release -H/home/aqtest/cmake-test -B/home/aqtest/cmake-test/build -G "Unix Makefiles"
[cmake] Not searching for unused variables given on the command line.
[cmake] LOWERCASEVAR=value
[cmake] -- Configuring done
[cmake] -- Generating done
[cmake] -- Build files have been written to: /home/aqtest/cmake-test/build
```

### Platform and Versions

- **Operating System**: Ubuntu
- **CMake Version**: 3.16.3
- **VSCode Version**: 1.51.1
- **CMake Tools Extension Version**: 1.5.3
- **Compiler/Toolchain**: Any

### Minimal project example to reproduce:

```
.vscode
|   cmake-kits.json
CMakeLists.txt
envsetup.sh
```

**cmake-kits.json**

```json
[
    {
        "name": "TestKit",
        "environmentSetupScript": "${workspaceFolder}/envsetup.sh"
    }
]
```

**CMakeLists.txt**

```cmake
cmake_minimum_required(VERSION 3.10)

project(TEST)

execute_process(COMMAND "bash" "-c" "env | grep -i lowercasevar")
```

**envsetup.sh**

```bash
#!/bin/bash
export lowercasevar=value
```

Steps:

1. Select kit as `TestKit`
2. Configure.

**Comment by andreeis** [comment]
Thank you for opening this report. We will fix this.

**Comment by bobbrow** [comment]
The fix for this issue is available in 1.10 "pre-release".  You can help us validate it by:
1. Opening the extensions panel in VS Code
2. Selecting the CMake Tools extension
3. Clicking the "Switch to Pre-release Version" button

![image](https://user-images.githubusercontent.com/12818240/156218538-7bce4a1f-4fc9-43cc-afb0-7f423aa5980e.png)
